### PR TITLE
fix(circleci): update ci

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

**Comment by linuxbandit** [comment]
big change (in the ci file), what is it about?

also it contains things which are not (apparently?) related to the ci, how come they are here?

File: package.json [hunk 1]
```diff
@@ -40,12 +37,6 @@
     "cli": "node --experimental-repl-await -r ./cli.js",
     "open-coverage": "open-cli coverage/lcov-report/index.html"
   },
-  "husky": {
```
Comment (linuxbandit) [inline package.json:43]: how is husky launched now? i saw the files above, i mean I don't understand how they are launched now

**Comment by WikiRik** [comment]
> big change (in the ci file), what is it about?
> 
> also it contains things which are not (apparently?) related to the ci, how come they are here?

These changes are the same as I did in statutory. It indeed is also updates not related to CI, like upgrade Node from 12 to 14. I put it in one PR on statutory, but maybe a different name is better. Any ideas?

File: package.json (hunk 1 above)
Comment (WikiRik) [reply package.json:43]: If you have husky installed locally it runs when you do a commit.

**Pull request merged**